testdata/cgo/atomic.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+// #include "atomic.h"
+// #cgo CFLAGS: -std=c11
```

Comment:
Clang defaults to `-std=gnu17`, so I don't think this is needed. Source: https://clang.llvm.org/docs/UsersManual.html#differences-between-various-standard-modes